src/components/Home.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="restaurants">
+    <div class="restaurants__img"></div>
+    <h1 class="restaurants__header">Najlepsze restauracje w Poznaniu</h1>
+    <ul class="restaurants__list">
+      <li
+        class="restaurants__item"
+        v-for="restaurant in restaurants"
+        :key="
+     restaurant.restaurant.id"
+      >
+        <a :href="restaurant.restaurant.events_url">{{restaurant.restaurant.name}}</a>
+        Ocena: {{restaurant.restaurant.user_rating.aggregate_rating}}
+        Kuchnia: {{restaurant.restaurant.cuisines}}
+        <router-link :to="`/restaurant/${restaurant.restaurant.id}`">Więcej</router-link>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import store from '../store'
+
+export default {
+  name: 'Home',
+  computed: {
+    restaurants () {
+      if (store.restaurants.length > 0) {
+        const sorted = store.restaurants.sort(
+          (a, b) =>
+            parseFloat(b.restaurant.user_rating.aggregate_rating) -
+            parseFloat(a.restaurant.user_rating.aggregate_rating)
```

Comment:
JavaScript weak-typed, it "likes" changing types depending on how a value is used. Here, you take "any" type and want to get a number. An no-op operation for numbers that isn't allowed for strings is one-arg plus (`+n`). It's also used in asm.js, a highly optimized subscript of JavaScript, [to denote a `double` type](http://asmjs.org/spec/latest/#parameter-type-annotations).